MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -42,6 +43,7 @@ public enum TargetPlatform
         /// Linux-based PCs
         /// (MonoGame)
         /// </summary>
+        [Obsolete("This platform is obsolete, use DesktopGL instead")]
         Linux,
```

Comment:
It was getting caused because DesktopGL is bellow Linux, moving it upwards fixes it.